restapi.go

```diff
@@ -1084,7 +1096,7 @@ func (s *Session) ChannelMessage(channelID, messageID string) (st *Message, err
 // messageID : the ID of a Message
 func (s *Session) ChannelMessageAck(channelID, messageID string) (err error) {
 
-	_, err = s.request("POST", EndpointChannelMessageAck(channelID, messageID), "", nil)
+	_, err = s.request("POST", EndpointChannelMessageAck(channelID, messageID), "", nil, 0)
```

Comment:
If my memory serves right reason it was not using Request was because that function adds the application/json content type, which the ack endpoint did not like, although atm the ack in discordgo is completely broken i think since it does stuff with tokens (haven't looked into it too much)